greedy_algorithms/dijkstra.cpp

```diff
@@ -1,40 +1,67 @@
-#include <limits.h>
+/**
+ * @file
+ * \brief Implementation of Dijkstra's Algorithm
+ * \details
+ * Dijkstra's Algorithm is a greedy solution for the single-source shortest path problem in a graph whose edges have integer weights.
+ * This solution takes in input to build the graph and outputs distances to nodes from a selected source node.
+ * @author arpanjain97
+ * @author christianbender
+ * @author ali-mir
+*/
+
+#include <climits>
```

Comment:
`climits` provides interface to C function from `limits.h`
`limits` header provides with `std::numeric_limits` namespace with C++ members

Reply:
You can retain using INT_MAX variable with limits. but please ensure the code executes correctly. 
Thanks.